deepchecks/tabular/checks/train_test_validation/single_feature_contribution_train_test.py

```diff
@@ -119,22 +122,34 @@ def run_logic(self, context: Context) -> CheckResult:
 
         return CheckResult(value=ret_value, display=display, header='Single Feature Contribution Train-Test')
 
-    def add_condition_feature_pps_difference_not_greater_than(self: FC, threshold: float = 0.2) -> FC:
+    def add_condition_feature_pps_difference_not_greater_than(self: FC, threshold: float = 0.2,
+                                                              include_negative_diff: bool = True) -> FC:
         """Add new condition.
 
         Add condition that will check that difference between train
         dataset feature pps and test dataset feature pps is not greater than X.
 
         Parameters
         ----------
-        threshold : float , default: 0.2
-            train test ps difference upper bound.
+        threshold: float, default: 0.2
+            train test pps difference upper bound.
+        include_negative_diff: bool, default True
+            This parameter decides whether the condition checks the absolute value of the difference, or just the
+            positive value.
+            The difference is calculated as train PPS minus test PPS. This is because we're interested in the case
+            where the test dataset is less predictive of the label than the train dataset, as this could indicate
+            leakage of labels into the train dataset.
         """
 
         def condition(value: t.Dict[Hashable, t.Dict[Hashable, float]]) -> ConditionResult:
+
+            diff_dict = copy(value['train-test difference'])
```

Comment:
why not just put abs in the if abs(pps_diff)

Reply:
because then I need to duplicate the "failed_features" (dict which is much longer) in order to enable abs or not abs

Reply:
why would you need to copy it?